packages/e2e-test-utils/src/insert-block.js

```diff
@@ -2,7 +2,6 @@
  * Internal dependencies
  */
 import { searchForBlock } from './search-for-block';
-import { getAllBlocks } from './get-all-blocks';
 
 /**
  * Opens the inserter, searches for the given term, then selects the first
```

Comment:
Noting that the changes in 4d231153495f5262c20b971ae145fe7fbd19d8a8 included a revision to the function documentation: "It then waits briefly for the block list to update.". It wasn't reverted here.